src/pwa-buildpack/project-setup/create-configuration-files/index.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Create configuration files
+---
+
+In the [previous topic], you linked your project to the Magento backend store.
+In this topic, you will create the configuration files for your development environment.
+
+## Create the Babel configuration file
+
+Create a `.babelrc` file in your theme’s root directory with the following content:
+
+``` javascript
+{
+    "plugins": [
+        "syntax-jsx",
+        "transform-class-properties",
+        "transform-object-rest-spread",
+        "transform-react-jsx"
+    ]
+}
+```
+
+This configuration installs the following [Babel] plugins:
+
+* [`syntax-jsx`] and [`transform-react-jsx`] - Transform [JSX] code into backwards-compatible JavaScript. 
+  JSX allows you to write React components using a syntax structure similar to HTML.
+    
+  See [Introducing JSX] for more information.
+* [`transform-class-properties`] - Transforms class properties and lets you use the following syntax:
+  ``` javascript
+  class Example {
+    anInstanceProperty = 52;
+  }
+  ```
+* [`transform-object-rest-spread`] - Transforms the rest and spread properties for objects to let you use the following syntax:
+  ``` javascript
+  const config = { ...defaultConfig, ...passedConfig };
+  ```
+
+In general, these plugins provide convenience and produces cleaner code in your project.
+
+## Create the local environment variables file
+
+Create a `.env` file and assign values to the following environment variables:
+
+* `MAGENTO_BACKEND_DOMAIN` - Your local Magento store’s host and port.
+* `MAGENTO_PATH` - The absolute path to the root directory of your local Magento store
+* `MAGENTO_BACKEND_PUBLIC_PATH` - Your theme’s vendor and name. The locale must be `en_US` for now.
```

Comment:
This should be set to `/`. We can comment that "PWA files are served from root during development, but this environment variable can later be used to simulate a deployed static path."